topology/probes/istio/graph.go

```diff
@@ -22,8 +22,20 @@
 
 package istio
 
+import (
+	"github.com/skydive-project/skydive/topology/graph"
+)
+
 const (
 	// Manager is the manager value for Istio
 	Manager      = "istio"
 	detailsField = "Istio"
 )
+
+func newEdgeMetadata(name string) graph.Metadata {
```

Comment:
I would rather place this function under k8s with the signature `NewEdgeMetadata(manager, name string)`

Reply:
DONE